keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/annealing/ResourcePersister.kt

```diff
@@ -42,9 +42,11 @@ class ResourcePersister(
     return if (diff.hasChanges()) {
       log.debug("Resource {} updated: {}", normalized.metadata.name, diff.toDebug(normalized.spec, existing.spec))
       normalized
-        .also(resourceRepository::store)
-        .also { resourceRepository.appendHistory(ResourceUpdated(it, diff.toUpdateJson(normalized.spec, existing.spec), clock)) }
-        .also { queue.scheduleCheck(it) }
+        .also {
+          resourceRepository.store(it)
+          resourceRepository.appendHistory(ResourceUpdated(it, diff.toUpdateJson(it.spec, existing.spec), clock))
+          resourceRepository.markCheckDue(it)
```

Comment:
Instead of explicitly checking a resource after an update I just bump it to the front of the line.